src/main/java/uk/nhs/hee/trainee/details/service/TrainingNumberGenerator.java

```diff
@@ -126,6 +126,12 @@ private String getParentOrganization(ProgrammeMembershipDto programmeMembership)
       default -> null;
     };
 
+    //override for military trainees
+    if (programmeMembership.getProgrammeMembershipType() != null
+        && programmeMembership.getProgrammeMembershipType().equalsIgnoreCase("MILITARY")) {
+      parentOrganization = "TSD";
+    }
```

Comment:
It's an enum on the TIS side, so it should never be anything other than `MILITARY` which lets us simplify to `Objects.equal(pmType, "MILITARY")`